Sources/SkipUI/SkipUI/Containers/Navigation.swift

```diff
@@ -146,7 +146,9 @@ public struct NavigationStack<Root> : View where Root: View {
         Scaffold(
             modifier: modifier,
             topBar: {
-                guard !isRoot || !title.value.isEmpty else {
+                let topLeadingItems = toolbarItems.filterTopBarLeading()
+                let topTrailingItems = toolbarItems.filterTopBarTrailing()
+                guard (!isRoot || !title.value.isEmpty) && !(topLeadingItems + topTrailingItems).isEmpty else {
```

Comment:
Do you mind changing this to:

guard !isRoot || !title.value.isEmpty || !topLeadingItems.isEmpty || !topTrailingItems.isEmpty 

That way the logic is bit simpler and also we don't have the overhead of combining the arrays just to see if either one is empty.

Reply:
No problem and makes sense, updated